src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -704,6 +707,21 @@ namespace AppInstaller::Runtime
         return wil::test_token_membership(nullptr, SECURITY_NT_AUTHORITY, SECURITY_BUILTIN_DOMAIN_RID, DOMAIN_ALIAS_RID_ADMINS);
     }
 
+    // Determines whether developer mode is enabled.
+    bool IsDevModeEnabled()
+    {
+        const auto& devModeSubKey = Registry::Key::OpenIfExists(HKEY_LOCAL_MACHINE, s_DevModeSubkey, 0, KEY_READ|KEY_WOW64_64KEY);
+        const auto& devModeEnabled = devModeSubKey[s_AllowDevelopmentWithoutDevLicense];
+        if (devModeEnabled.has_value())
+        {
+            return devModeEnabled->GetValue<Registry::Value::Type::DWord>();

```

Comment:
```suggestion
            return devModeEnabled->GetValue<Registry::Value::Type::DWord>() == 1;
```
The only value that enables it is 1.

Reply:
Changed to suggested